frontend/packages/topology/src/components/svg/SvgBoxedText.tsx

```diff
@@ -45,6 +46,7 @@ const SvgBoxedText: React.FC<SvgBoxedTextProps> = ({
   y = 0,
   kind,
   typeIconClass,
+  typeIcon,
```

Comment:
can't we use the existing prop we have here i.e `typeIconClass` instead of introducing a new one?

cc @sahil143

Reply:
typeIconClass accepts svg & in this case, we want to pass the react element